ibis/backends/tests/test_map.py

```diff
@@ -186,6 +186,182 @@ def test_literal_map_getitem_broadcast(backend, alltypes, df):
     backend.assert_series_equal(result, expected)
 
 
+@pytest.fixture(
```

Comment:
These don't need to be a fixture that returns its value, it can be a `pytest.mark.parametrize` and directly decorate the test function.

You can also do this if the goal was to give the fixtures a name:

```python
keys = pytest.mark.parametrize("keys", [...])
values = pytest.mark.parametrize("values", [...])
```